apps/app/src/components/AdminPanel/AdminPanel.tsx

```diff
@@ -52,7 +45,7 @@ export const AdminPanel = ({ page, technology, levels, status }: AdminPanelProps
 						total={data.data.meta.total}
 						getHref={(i) => ({
```

Comment:
Możesz to nazwać `page` i skrócić wtedy:
```suggestion
						getHref={(page) => ({
```